CHANGELOG.md

```diff
@@ -68,6 +68,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 - (deps) [#2967](https://github.com/evmos/evmos/pull/2967) Bump CometBFT to `v0.38.15`.
 - (precompiles) [#2943](https://github.com/evmos/evmos/pull/2943) Add WERC-20 precompile.
 - (precompiles) [#2966](https://github.com/evmos/evmos/pull/2966) Add safety check that ERC-20 precompiles cannot receive funds.
+- (evm) [#3000](https://github.com/evmos/evmos/pull/3000) Move population of access list with precompile addresses into EVM hook.
```

Comment:
Should we move this to breaking? It's going to change the used gas for access list transactions using the erc20 dynamic precompiles

Reply:
You're right!